.circleci/config.yml

```diff
@@ -50,7 +56,7 @@ jobs:
           command: script/bootstrap
       - run:
           name: Update
-          command: script/update --clean -t $TARGET_ARCH
+          command: script/update --clean -t $TARGET_ARCH --use-bundled-sccache
```

Comment:
Somehow missed it.
I thought we agreed on not using cache for Release builds.

Reply:
@alexeykuzmin why would we not use cache for Release builds???

Reply:
I mean it doesn't really gain us much if we only use this for shared builds.  We still end up having to wait for the static builds.  Given that we are using a cache that only the CI has write access to, I'm not sure why we wouldn't use it for Release builds.

Reply:
@jkleinsc For maximum safety, release builds should be generated in a 100% deterministic and isolated way.  Generating them without cache is what everyone was 👍 on from memory

Reply:
Alright then, I changed it to just run for shared builds.